lib/puppet/vendor/pathspec/lib/pathspec/spec.rb

```diff
@@ -1,4 +1,4 @@
-class Spec
+class BSpec
```

Comment:
That shouldn't have made it into the PR. The reason for the rename is that the module_tool tests doesn't run when a class is present that is named `Spec` because the tests introduce a module with that name. I pushed commit where that rename is reverted.